README.md

```diff
@@ -109,6 +109,31 @@ in {
 }
 ```
 
+### Integrating with HomeManager
```

Comment:
I generally use "Home Manager" (with a space) in documentation when I refer to the project as a whole and "home-manager" for the command line tool.
```suggestion
### Integrating with Home Manager
```